vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/StaffSearchSlideTextBlock.java

```diff
@@ -0,0 +1,37 @@
+package edu.asu.diging.vspace.core.model.impl;
+
+import java.util.List;
+import java.util.Map;
+
+public class StaffSearchSlideTextBlock {
+    
+    private List<Slide> slidesWithMatchedTextBlock;
+
+    private Map<String, String> slideTextFirstImage;
```

Comment:
bad variable name: `slideTextFirstImage` sounds like one object, but it's several and it's a map.